sim/visuals/microbit.ts

```diff
@@ -822,7 +887,7 @@ namespace pxsim.visuals {
         private buttonsOuter: SVGElement[];
         private pins: SVGElement[];
         private pinGradients: SVGLinearGradientElement[];
-        private pinTexts: SVGTextElement[];
+        private pinTexts: pinTexts;
```

Comment:
are you sure about this?

Reply:
I needed a way to get the pinText-Element by `pin.id` instead of `index`. Therefore I created the new object type.

I'm not professional in TypeScript, so is there a better solution or do I just shouldn't name the type like the variable?
Other ways I tried I got `Element implicitly has an 'any' type because expression of type 'string' can't be used to index {}`-Errors

Reply:
Ah, I could do this without defining a new type and use number instead of string for the key. Just pushed an update.